app/models/accommodation.rb

```diff
@@ -47,7 +47,7 @@ def self.filter_holds(admin)
 
   def get_label
     if (self.building.building_type.label.eql?('Cabin') || self.building.building_type.label.eql?('Lodge'))
-      return "#{self.building.label} #{self.label}"
+      return "#{self.label}"
```

Comment:
Yeah, I tried to be too slick with the labels last year and just moved to having everything in the accommodation this year... obviously missed some things in the process.